### ISSUE TITLE
Leechblock not working for Firefox widget

### ISSUE THREAD

**Issue by clegergh** [proginosko/LeechBlockNG#446]
When I use the widget search (perhaps only when the app was killed) blocking websites does not actually work, I can freely browse YouTube.

**Comment by clegergh** [comment]
Update: yes, when the app is killed and a new tab is created, leech block is bypassed

**Comment by proginosko** [comment]
Is this Firefox for Android?

**Comment by clegergh** [comment]
Yes omg sorry, yeah, Firefox for android.
On May 19, 2024, at 22:08, James Anderson ***@***.******@***.***>> wrote:

Is this Firefox for Android?

**Comment by proginosko** [comment]
This looks like a repeat of issue #265. Is "widget" another name for "web app"?

**Comment by clegergh** [comment]
No, it's like with Firefox you made the search bar widget. It sits on your home screen and you can click it and it will instantly make a new tab within Firefox.

When a new tab is created and Firefox was not previously opened, it bypasses leechblock.

It could be a problem with Firefox itself, but it would be nice if leechblock checked each website every 5 seconds or so to double down on new tabs (maybe as an opt in thing).
On May 20, 2024, at 10:39, James Anderson ***@***.******@***.***>> wrote:

This looks like a repeat of issue #265<https://github.com/proginosko/LeechBlockNG/issues/265>. Is "widget" another name for "web app"?
